### PR TITLE
1st time improvements

### PR DESCRIPTION
**Pull request by alankar63**

### CODE REVIEW

File: sorted_rating.sh [hunk 1]
```diff
@@ -1,30 +1,48 @@
-echo enter directory which contains movie folder
-read dir
-final_rate.txt
-for f in $dir/movies/*
+: '
```
Comment (vishesh92) [inline sorted_rating.sh]: Missed the shebang.

Comment (vishesh92) [inline sorted_rating.sh]: and what is `:` in the starting?

File: sorted_rating.sh [hunk 2]
```diff
@@ -1,30 +1,48 @@
...
+2.2. Movies with same rating are sorted with name 
+3.   assumptions :
+3.1  directory to be provided after bash command
+4.   Output file :final_rate.txt
+---------------------------------------------------------------------------
+'
+
+touch final_rate.txt
+> final_rate.txt
+#final_rate needs to be emptied
```
Comment (vishesh92) [inline sorted_rating.sh]: This comment should be above the statement. And leave a space after `#`.

File: sorted_rating.sh [hunk 3]
```diff
@@ -1,30 +1,48 @@
...
+
+if [ -z $1 ];
+then
+   echo 'Directory not provided. Searching in current directory'
+   dir='.'
+else
+   #echo 'Input Directory is' $1
```
Comment (vishesh92) [inline sorted_rating.sh]: code is generally not commented.

File: sorted_rating.sh [hunk 4]
```diff
@@ -1,30 +1,48 @@
...
+---------------------------------------------------------------------------
+'
+
+touch final_rate.txt
+> final_rate.txt
+#final_rate needs to be emptied
+
+if [ -z $1 ];
+then
+   echo 'Directory not provided. Searching in current directory'
```
Comment (vishesh92) [inline sorted_rating.sh]: indentation used here is 3 spaces.

File: sorted_rating.sh [hunk 5]
```diff
@@ -1,30 +1,48 @@
...
-   printf $data1 >>final_rate.txt
+   #rating kept in data
+   
+   printf $data1 >> final_rate.txt
    printf " " >> final_rate.txt
-   echo $oname >> final_rate.txt
+   echo $orig_name >> final_rate.txt
 
-done
+ done
```
Comment (vishesh92) [inline sorted_rating.sh]: This does not seem to be matching.

Comment (vishesh92) [inline sorted_rating.sh]: https://google.github.io/styleguide/shell.xml?showone=Indentation#Indentation

File: sorted_rating.sh [hunk 6]
```diff
@@ -1,30 +1,48 @@
...
-   cut -c-9 out.txt >  final.txt
-
-   read val <final.txt
-
-   curl -s http://www.imdb.com/title/$val/ >rating.html
-
-   data=$(grep "based on" rating.html|grep "[0-9]\.[0-9]" -o)
-  
+   curl -s  http://www.imdb.com/find\?ref_\=nv_sr_fn\&q\=$name\&s\=all | \
+   grep -E -0  '/tt\w+' |cut -c72- > out.txt
```
Comment (vishesh92) [inline sorted_rating.sh]: The format here is incorrect.
https://google.github.io/styleguide/shell.xml?showone=Line_Length_and_Long_Strings#Line_Length_and_Long_Strings

File: sorted_rating.sh (hunk 5 above)
Comment (alankar63) [inline sorted_rating.sh]: thats original name

File: sorted_rating.sh (hunk 3 above)
Comment (vaidik) [inline sorted_rating.sh]: Yes please get rid of this. Not good to leave commented code.

**Pull request merged**